README.md

```diff
@@ -2,8 +2,12 @@
+The Software Citation Tools is a package of tools written in javascript and made available through the npm javascript package manager. The main part of the package will be a library which will create a citation when given a link to a github repository. Further tools will be created on top of this library to make using this library easier and extend it's usefulness.
 
-At the core of the Software Citation Tools suite will be a package written in JavaScript using npm.  This package will provide functions which will take a GitHub repository URL and return a citation for the software compliant with the Software Citation Principles.  Stretch goals for this package may include accepting other types of software repositories as input, and accepting parameters that will change the format of the citation.  For example, parameters to generate long and short versions of a citation, or output for different citation standards.  The priority of stretch goals will be determined based on the needs of the scientific community.
+##About the Project
+The Software Citation Tools is a project proposed and overseen by Mozilla Science Lab via Abigaill Cabunoc Mayes and headed by senior project team at The Rochester Institute of Technology's Software Engineering Department. The purpose of Software Citation Tools is to provide some tools that use the ideas shown in the [Software Citation Principles](https://www.force11.org/software-citation-principles) paper by FORCE11 to make software citation easier and more cited in academia.
 
-Utilizing the citation package, two or more applications will be developed.  Two such applications might be a webapp interface to allow researchers to utilize the package conveniently, and a browser plugin to produce the citation when the user has navigated to a repo.
+##Contributing
+Even though the team heading the project is doing this as a Senior Project, anyone is free to contribute! The RIT Software Engineering department has given approval for anyone to help out in any way without effecting their grade. Please feel free to contribute in any way you want!
+
+Right now we're trying to build a set of requirements and features by getting responses to some [questions](https://github.com/mozillascience/software-citation-tools/issues?q=is%3Aissue+is%3Aopen+label%3ADiscussion) we made.
```

Comment:
+1 on this.  Wouldn't stop me from approving the PR, but something we should consider.  What do you all think of a "Current Action Items" or similar section below line 5, before the first heading?